src/SearchInput/SearchInput.tsx

```diff
@@ -124,51 +142,74 @@ export const SearchInput: FC<SearchInputProps> = ({
   )
 }
 
-const Container = styled.div<IContainer>`
+const Container = styled.div<IUsesOutline>`
   position: relative;
   width: 100%;
   background: ${theme.colors.white};
 `
 
+const InnerContainer = styled.div<IUsesOutline>`
+  ${({ outlined }) =>
+    outlined &&
+    `
+    position: relative;
+    border: 1px solid ${theme.colors.grey3};
+    border-radius: 8px;
+    padding: 10px;
+  `}
+`
+
 const Input = styled.input<ISearchInput>`
   display: block;
   border: none;
-  border-bottom: 1px solid;
-  border-color: ${theme.colors.grey4};
+  border-bottom: ${({ outlined }) =>
+    outlined ? 'none' : `1px solid ${theme.colors.grey4}`};
   outline: none;
-  color: ${theme.colors.blue7};
+  color: ${({ outlined }) =>
+    outlined ? `${theme.colors.grey8}` : `${theme.colors.blue7}`};
   font-size: 16px;
   height: 32px;
   width: 100%;
   box-sizing: border-box;
 
   &::placeholder {
-    color: ${theme.colors.grey4};
+    color: ${({ outlined }) =>
+      outlined ? theme.colors.grey8 : theme.colors.grey4};
   }
 `
 
+const IconContainer = styled.div`
+  position: absolute;
+  top: 50%;
+  right: 20px;
+  transform: translateY(-50%);
+`
+
 const ResultsContainer = styled.div<IResultsContainer>`
   box-sizing: border-box;
   overflow-y: hidden;
-  ${(p) => p.absolutePosition && 'position: absolute;'}
+  ${({ absolutePosition }) => absolutePosition && 'position: absolute;'}
   width: 100%;
-  visibility: ${(p) => (p.show ? 'visible' : 'hidden')};
+  visibility: ${({ show }) => (show ? 'visible' : 'hidden')};
+  ${({ outlined }) => outlined && 'left: 0px; top: 90%;'}
```

Comment:
any reason why this is needed? It's hard to understand styling changes without a video of the functionality of it